java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk

```diff
@@ -33,13 +33,18 @@ TFLITE_FLAGS := \
 # FLOAT16/COMPLEX64 for tensorflow-lite >= 2, and INT8/INT16 for tensorflow-lite >=1.13
 # GPU-delegate supported on tensorflow-lite >= 2
 # NNAPI-delegate supported on tensorflow-lite >= 1.14
+# XNNPACK-delegate supported on tensorflow-lite >= 2.3.0
 TFLITE_ENABLE_GPU_DELEGATE := false
 TFLITE_ENABLE_NNAPI_DELEGATE := false
+TFLITE_ENABLE_XNNPACK_DELEGATE := false
 TFLITE_EXPORT_LDLIBS :=
 
 ifeq ($(shell test $(TFLITE_VERSION_MAJOR) -ge 2; echo $$?),0)
 TFLITE_ENABLE_GPU_DELEGATE := true
 TFLITE_ENABLE_NNAPI_DELEGATE := true
+ifeq ($(shell test $(TFLITE_VERSION_MINOR) -ge 3; echo $$?),0)
+TFLITE_ENABLE_XNNPACK_DELEGATE := true
```

Comment:
This is FALSE if ```tf-lite == 3.0``` XD

Reply:
Because we maintain our tflite packages, IMO, that case could be handled in the future.